WcaOnRails/app/models/registration.rb

```diff
@@ -102,7 +107,7 @@ def waiting_list_info
 
   validate :must_register_for_gte_one_event
   private def must_register_for_gte_one_event
-    if events.empty?
+    if registration_events.length { |r| !r.marked_for_destruction? } == 0
```

Comment:
To what does the `marked_for_desctruction` correspond?
Is this whole call equivalent to `events.empty?`?